<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportProperties</class>
 <widget class="QWidget" name="ReportProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="url">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="reportName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Author:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>URL:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTextEdit" name="description">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="author">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Options:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabStorage">
      <attribute name="title">
       <string>Storage</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelRendering_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Storage module:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>298</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Storage Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="storageType">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QComboBox" name="storageModules">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>190</width>
              <height>0</height>
             </size>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="assignStorage">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>assign</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" rowspan="3" colspan="2">
           <widget class="QTreeWidget" name="storageList">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="itemsExpandable">
             <bool>false</bool>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="bDefaultStorage">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../script_editor/script_editor.qrc">
              <normaloff>:/images/validate.png</normaloff>:/images/validate.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QToolButton" name="bDeleteStorage">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../dataset_editor/dataset_editor.qrc">
              <normaloff>:/images/trash-empty.png</normaloff>:/images/trash-empty.png</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="2" rowspan="2">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>107</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Default:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QToolButton" name="bResetDefaultStorage">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../script_editor/script_editor.qrc">
              <normaloff>:/images/editdelete.png</normaloff>:/images/editdelete.png</iconset>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QLineEdit" name="leDefaultStorage">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Storage settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="storageHelperLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRenderer">
      <attribute name="title">
       <string>Renderer</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelRendering">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Rendering module:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="renderingModules"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="assignRenderer">
         <property name="text">
          <string>assign</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>238</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="5" rowspan="4">
        <widget class="QFrame" name="propertyEditorFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Assigned Renderer:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QLineEdit" name="rendererName">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QToolButton" name="bRendererDelete">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../dataset_editor/dataset_editor.qrc">
           <normaloff>:/images/trash-empty.png</normaloff>:/images/trash-empty.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLabel" name="lGlobalRenderer">
         <property name="text">
          <string>Global default module is in use</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>139</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPrinter">
      <attribute name="title">
       <string>Printer</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="labelPrinting">
         <property name="minimumSize">
          <size>
           <width>130</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Printing module:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="printModules"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="assignPrinter">
         <property name="text">
          <string>assign</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" colspan="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>238</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Assigned Printer:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QLineEdit" name="printerName">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QToolButton" name="bPrinterDelete">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../dataset_editor/dataset_editor.qrc">
           <normaloff>:/images/trash-empty.png</normaloff>:/images/trash-empty.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLabel" name="lGlobalPrinter">
         <property name="text">
          <string>Global default module is in use</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>139</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="5" rowspan="4">
        <widget class="QFrame" name="printerPropertyEditorFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Variables:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="4">
    <widget class="QTableWidget" name="variables">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <zorder>label</zorder>
  <zorder>description</zorder>
  <zorder>reportName</zorder>
  <zorder>variables</zorder>
  <zorder>author</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>tabWidget</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>url</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources>
  <include location="../script_editor/script_editor.qrc"/>
  <include location="../dataset_editor/dataset_editor.qrc"/>
 </resources>
 <connections/>
</ui>
